contracts/Overtime/LiveTradingProcessor.sol

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+// external
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "./SportsAMMV2.sol";
```

Comment:
okey if the compiler optimizes, but I guess it's better using an interface

Reply:
used the interface

Reply:
I would just import the interface for lower contract size